lib/logstash/outputs/elasticsearch/protocol.rb

```diff
@@ -98,12 +98,17 @@ def self.normalize_bulk_response(bulk_response)
 
       def bulk(actions)
         bulk_response = @client.bulk(:body => actions.collect do |action, args, source|
+          if action == "create_unless_exists"
```

Comment:
ah, for some reason I thought action was a formatted string. it isn't.

I do not know how to escape `Stud::Buffer`: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/lib/logstash/outputs/elasticsearch.rb#L36-L37